<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Safe Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="firstName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="phone">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="email">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="dob">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DOB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createRecordButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Record</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="middleName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lastName">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="secondLastName">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Second/Middle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Last/Paternal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Last/Maternal</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="nameSuffix">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>110</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>90</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Suffix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefix</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="namePrefix">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1222</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>namePrefix</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>middleName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>secondLastName</tabstop>
  <tabstop>nameSuffix</tabstop>
  <tabstop>phone</tabstop>
  <tabstop>email</tabstop>
  <tabstop>dob</tabstop>
  <tabstop>createRecordButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
